x-pack/plugins/ml/public/application/explorer/explorer_utils.ts

```diff
@@ -600,3 +614,51 @@ export function removeFilterFromQueryString(
 
   return newQueryString;
 }
+
+// Returns an object mapping job ids to source indices which map to geo fields for that index
+export async function getSourceIndicesWithGeoFields(
+  selectedJobs: Array<CombinedJob | ExplorerJob>,
+  dataViewsService: any
+): Promise<SourceIndicesWithGeoFields> {
+  const sourceIndicesWithGeoFieldsMap: SourceIndicesWithGeoFields = {};
+  // Go through selected jobs
+  if (Array.isArray(selectedJobs)) {
+    await asyncForEach(selectedJobs, async (job) => {
+      let sourceIndices;
+      let jobId: string;
+      if (isCombinedJob(job)) {
```

Comment:
This guard might be better as a `isExplorerJob` check instead.
All jobs now contain `datafeed_config` and so this is really a check for is this a normal ML job.
At some point in the future we'll need to get rid or the `CombinedJob` type.

Reply:
Replaced in 8138a2d6498c77580a6641bc1b11afa311fee4af